### PR TITLE
Add SOM VM definition

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

File: krun/vm_defs.py [hunk 1]
```diff
@@ -740,3 +740,25 @@ def run_exec(self, entry_point, iterations, param, heap_lim_k,
...
+    def run_exec(self, entry_point, iterations, param, heap_lim_k,
+                 stack_lim_k, key, key_pexec_idx, force_dir=None,
+                 sync_disks=True):
+
+        benchmark = key.split(":")[0]
+        script_dir = os.path.join(BENCHMARKS_DIR, benchmark)
+        script_path = self._get_benchmark_path(benchmark, entry_point,
+                                               force_dir=force_dir)
+
+        args = [self.vm_path] + ['--cp'] + [":".join(self.classpath)] + self.extra_vm_args + \
```
Comment (ltratt) [inline krun/vm_defs.py:760]: Isn't this the same as `args = [self.vm_path, '--cp', ":".join(...)] + self.extra_vm_args + ...` ?

Comment (jacob-hughes) [reply krun/vm_defs.py:760]: You're right. It seems I've forgotten how to python.

Comment (jacob-hughes) [reply krun/vm_defs.py]: Fixed: 3add547

**Comment by jacob-hughes** [comment]
Squashed

**Review by ltratt — APPROVED** [review]

**Pull request merged**